mteb/tasks/Retrieval/fra/FQuADRetrieval.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class FQuADRetrieval(AbsTaskRetrieval):
+    _EVAL_SPLITS = ["test_hasAns", "valid_hasAns"]
+
+    metadata = TaskMetadata(
+        name="FQuADRetrieval",
+        description="This dataset has been built from the French SQuad dataset.",
+        reference="https://huggingface.co/datasets/manu/fquad2_test",
+        dataset={
+            "path": "manu/fquad2_test",
+            "revision": "5384ce827bbc2156d46e6fcba83d75f8e6e1b4a6",
+        },
+        type="Retrieval",
+        category="s2p",
+        eval_splits=_EVAL_SPLITS,
+        eval_langs=["fra-Latn"],
+        main_score="map",
+        date=None,
+        form=None,
+        domains=["Encyclopaedic"],
+        task_subtypes=None,
+        license="apache-2.0",
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=[],
+        text_creation="created",
+        bibtex_citation="""@misc{faysse2024croissantllm,
+                      title={CroissantLLM: A Truly Bilingual French-English Language Model}, 
+                      author={Manuel Faysse and Patrick Fernandes and Nuno M. Guerreiro and António Loison and Duarte M. Alves and Caio Corro and Nicolas Boizard and João Alves and Ricardo Rei and Pedro H. Martins and Antoni Bigata Casademunt and François Yvon and André F. T. Martins and Gautier Viaud and Céline Hudelot and Pierre Colombo},
+                      year={2024},
+                      eprint={2402.00786},
+                      archivePrefix={arXiv},
+                      primaryClass={cs.CL}
+                }""",
+        n_samples={"test": 400, "validation": 100},
+        avg_character_length={"test": 700, "validation": 700},
+    )
+
+    def load_data(self, **kwargs):
```

Comment:
please don't overwrite the load_data function unless required. Please use the `dataset_transform()` function instead.

Reply:
Hello ! About that, the `dataset_transform` function is not called in the AbsTaskRetrieval `load_data` function, which is itself very restrictive in the input format required (needs 2 separate splits for corpus, queries, etc etc)...

Even by fixing the AbsTaskRetrieval to call the transform, it is impossible not to override `load_data` given the dataset at hand here, so I kept it that way for the moment.

Reply:
Ahh yes, you are correct. Then this is perfectly fine